tests/python/gpu/test_gluon_model_zoo_gpu.py

```diff
@@ -105,6 +107,7 @@ def test_training():
     # TODO(zhengda) mobilenet can't pass this test even without MKLDNN.
     all_models = ['resnet18_v1', 'densenet121']
 
+    mx.random.seed(1)
```

Comment:
please don't use mx.random.seed due to the introduction of the @with_seed() decorator